tfjs-react-native/src/version.ts

```diff
@@ -0,0 +1,5 @@
+/** @license See the LICENSE file. */
+
+// This code is auto-generated, do not modify this file!
+const version = '0.2';
+export {version};
```

Comment:
re-export `version` from `index.ts` to be consistent with the other packages

Reply:
Done!